src/layouts/About.js

```diff
@@ -31,28 +36,28 @@ const Legal = () => {
                 className="fdb-icon"
                 src="https://cdn.jsdelivr.net/gh/froala/design-blocks@master/dist/imgs/icons/github.svg"
               />
-              <h1>{t('contributeTitle')}</h1>
-              <p className="lead">
+              <h2 className="h1">{t('contributeTitle')}</h2>
+              <LeadParagraph className="lead">
                 <Trans i18nKey="about:contributeText">
                   CovPy es libre y de código abierto. Puedes encontrar el código
                   en GitHub. Las contribuciones son bienvenidas! También puedes
                   simplemente
-                  <a href="https://www.github.com/penguin-academy/covid-19-py/issues">
+                  <ColoredA href="https://www.github.com/penguin-academy/covid-19-py/issues" rel="noopener noreferrer">
                     informar de un error
-                  </a>
+                  </ColoredA>
                   o solicitar una característica allí. report a bug or request a
                   feature there.
                 </Trans>
-              </p>
+              </LeadParagraph>
               <p className="h3 mt-4">
-                <a href="https://www.github.com/penguin-academy/covid-19-py">
-                  {t('learnMore')} <i className="fas fa-angle-right" />
-                </a>
+                <ColoredA href="https://www.github.com/penguin-academy/covid-19-py" rel="noopener noreferrer" alt={t("repo")}>
+                  {t('learnMore')} <i className="fas fa-angle-right" aria-hidden="true" />
+                </ColoredA>
               </p>
             </div>
             <div className="col-10 col-sm-6 m-auto col-md-4 pt-4 pt-md-0">
               <img
-                alt="chat"
+                alt=""
```

Comment:
In instances like this one, where the image serve decorative purposes, an empty alternative text is used instead, since they don't contain content nor functionality.